ParisBNF/abb/BNFabb88.xml

```diff
@@ -0,0 +1,303 @@
+                                <desc>Careful, irregular; by a good scribe. <persName ref="PRS1071dAbbadi"/> describes the handwriting as <foreign xml:lang="fr">ʻguịlḥ moderneʼ</foreign>.
+                                </desc>
+                            </handNote>
+                        </handDesc>
+                        <decoDesc>
+                            <decoNote xml:id="d1" type="miniature">
+                                <locus target="#2v"/>
+                                <desc><ref type="authFile" corresp="AT1042HagiographicScene">: <persName ref="PRS9848Walatta"/></ref>Black and white image does not allow any certain interpretation of the miniature.</desc>
+                            </decoNote>
+                            <decoNote xml:id="d2" type="miniature">
+                                <locus target="#3v"/>
+                                <desc><ref type="authFile" corresp="AT1042HagiographicScene"><persName ref="PRS9848Walatta"/>Walatta Ṗeṭros taming the hippopotamuses on Lake Tana.</ref></desc>
+                                    <note>Interpretation of the miniature is taken from
+                                    <bibl>
+                                        <ptr target="bm:BoscWion2010Abbadie"/>
+                                        <citedRange unit="page">90</citedRange>
+                                    </bibl>, containing a colorful reproduction of the miniature.
+                                    </note>
+                                
+                            </decoNote>
+                            <decoNote xml:id="d3" type="miniature">
+                                <locus target="#4r"/>
+                                <desc><ref type="authFile" corresp="AT1042HagiographicScene"><persName ref="PRS9848Walatta"/></ref>Black and white image does not allow any certain interpretation of the miniature.</desc>
+                            </decoNote>
+                        </decoDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus target="1ra"/>
+                                    <desc type="GuestText">
+                                        <title type="incomplete">Miracle of Walatta Ṗeṭros</title> 
+                                    </desc>
+                                </item>
+                                <item xml:id="a2">
+                                    <locus from="1va" to="2ra"/>
+                                    <desc type="GuestText">
+                                        <title type="complete" ref="LIT5694LukeSalam">Luke 1:26-38 and salām-hymns</title>
+                                    </desc>
+                                </item>
+                                <item xml:id="a3">
+                                    <locus target="#5r"/>
+                                    <desc type="Inventory"/>
```

Comment:
really Inventory? Inventory is a list of books of a library or items in a house etc.
If you mean that it is a handwritten insertion of a shelfmark then it is not an `a` addition but an `e` extra and surely not an Inventory